docs/proposals/multi-arch-image-mgmt.md

```diff
@@ -55,19 +49,50 @@ To create a multi-arch image, ORAS users have to manually compose an OCI image i
 
 Users are not able to install or use `docker buildx` or even no `docker` and its daemon service in some strict environments. Users need to seek for an alternative tool to create a multi-arch image without relying on `docker` or `docker buildx`.
 
-## Proposed solution
+## Scenarios
+
+### Create a multi-arch image in local filesystem
+
+Bob needs to create a multi-arch image in an air-gapped environment with strict isolation, meaning all resources must reside in the local filesystem, with no access to a container registry. Each architecture-specific image manifest has already been created and stored locally. Bob wants to use the `oras` client to assemble a multi-arch image by creating an OCI image index that references these existing arch-specific images. Once the multi-arch image is assembled, Bob will verify and test it locally to ensure correctness.
+
+### Create a multi-arch image from local and registry
+
+The user Bob wants to create a multi-arch image by combining architecture-specific images stored in both a local filesystem and a remote registry. Bob has already created some arch-specific image manifests and stored them locally, while others exist in a container registry. He wants to use the `oras` client to create a multi-arch image using an OCI image index, incorporating both local and remote images. Once created, Bob will push the final multi-arch image to a remote registry for distribution and testing.
+
+### Create a multi-arch image from registry only
+
+Bob wants to create a multi-arch image using arch-specific images stored in a remote container registry. Each architecture-specific image has already been pushed to the registry. Bob aims to use the `oras` client to assemble these remote images into a multi-arch image using an OCI image index, without downloading them to the local filesystem. After successfully creating the multi-arch image, Bob will push the final multi-arch to the registry for further use.
```

Comment:
Why is it always "Bob" in all scenarios? 😂

Reply:
Just wanted to keep the sample user name easier to memory